### ISSUE TITLE
Python < 3.8 compatibility 

### ISSUE THREAD

**Issue by pbelange** [xsuite/xsuite#145]
The f-string formatting  (ex: `f'(i_aper_1=)'`) used in some files return an error when using python 3.7.9 (or anything < 3.8) such as :

```python
 File "/home/pbelange/abp/CALS2Mask/py_wireDAQ/xtrack/xtrack/loss_location_refinement/__init__.py", line 6, in <module>
    from .loss_location_refinement import LossLocationRefinement

  File "<fstring>", line 1
    (i_aper_1=)
             ^
SyntaxError: invalid syntax
```

This is inconvenient given that the `acc-py` (https://pylhc.github.io/packages/development/howto_venv/) distribution used for SPARKS and NXCALS uses python 3.7.9 with no easy method of upgrading the distribution. Maybe the f-string formatting could be adjusted at no cost to improve compatibility?

**Comment by giadarol** [comment]
Feel free to make a pull request to ensure python 3.7 compatibility. I would say that it is not too difficult

**Comment by giadarol** [comment]
Merged on v0.18.0
